test/results.el

```diff
@@ -445,7 +445,7 @@ Wednesday   5 July 2017
  Scheduled earlier
   ambition:   Sched. 1x:  TODO [#A] Skype with president of Antarctica                             :universe:ambition:world::meetings:
 " "5db2acc8c5a09927fa8b6246b9480d69" "Headlines with TAGS match: Emacs
-Press ‘C-u r’ to search again
+Press `C-u r' to search again with new search string
```

Comment:
Unfortunately, differences in Org versions cause minor changes like this to appear in test results.  Please be sure that the changes in the results file only result from the changes in the PR's code.  You can use, e.g. Magit to discard unrelated hunks.

Reply:
Good idea. Also, a somewhat confusing work - I have eliminated all problems but one in results.el.